react/src/pages/admin/courses/CourseManagementPage.jsx

```
@@ -0,0 +1,79 @@
+                })
+                .catch(error => {
+                    console.error('Error deleting course:', error);
+                });
+        }
+    };
+
+    return (
+        <div className={`container ${styles.courseManagement}`}>
+            <div className="d-flex justify-content-between align-items-center">
+                <h1 className="text-primary">The List of Courses</h1>
+                <button onClick={() => navigate('/admin/courses/create')} className="btn btn-success">Create New Course</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)